spec/support/vcr_cassettes/forms/200_all_forms.yml

```diff
@@ -0,0 +1,345 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://dev-api.va.gov/services/va_forms/v0/forms?query
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Accept:
+      - application/json
+      Content-Type:
+      - application/json
+      User-Agent:
+      - Vets.gov Agent
+      Apikey:
+      - 5sBmaAWUXwjUtG1qO7yZI6bAtYpg9KPH
```

Comment:
Im assuming this is your test apikey? You should replace this with a fake one

Reply:
That is an excellent idea. I'm gonna replace it, but also have it regenerated.